.github/workflows/release.yml

```diff
@@ -35,8 +35,8 @@ jobs:
         id: import_gpg
         uses: crazy-max/ghaction-import-gpg@v5.0.0
         with:
-          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
-          passphrase: ${{ secrets.PASSPHRASE }}
+          gpg_private_key: ${{ secrets.GPG_ARTIFACT_SIGNING_KEY }}
```

Comment:
same question on key change compatibility here

Reply:
The sha256 sum file is being signed and a signature is created. People can download our publically accessible key and verify like this:
```
gpg --verify terraform-provider-awsapigateway_0.2.0_SHA256SUMS.sig terraform-provider-awsapigateway_0.2.0_SHA256SUMS
```
If there's documentation to verify our distribution, we may need to modify that, but I don't think anything else is needed here.

@ravisingal thoughts?